src/pms/sensors/plantower/pmsx003.py

```diff
@@ -13,6 +13,7 @@
 
 ALIASES = ("PMS1003", "G1", "PMS5003", "G5", "PMS7003", "G7", "PMSA003", "G10")
 
+PREHEAT = 10  # 10 seconds
```

Comment:
Since this observation is only for PMSA003, would it make more sense to create a `pmsa003.py` module to keep the change isolated to that sensor?

Reply:
It might be mentioned only on the `PMSA003`, but it seems to me that this should affect all Plantower sensors that support `sleep`/`wake` commands.